doc/stdlib/test.rst

```diff
@@ -1,83 +1,188 @@
-*****************************
- Unit and Functional Testing
-*****************************
+**************
+ Unit Testing
+**************
 
-Testing Base Julia
-------------------
+.. module:: Base.Test
+
+The ``Base.Test`` module provides simple *unit testing* functionality.
+Unit testing is a way to see if your code is correct by checking that
+the results are what you expect. It can be helpful to ensure your code
+still works after you make changes, and can be used when developing as
+a way of specifying the behaviors your code should have when complete.
 
-Julia is under rapid development and has an extensive test suite to
-verify functionality across multiple platforms. If you build Julia
-from source, you can run this test suite with ``make test``. In a
-binary install, you can run the test suite using ``Base.runtests()``.
```

Comment:
It's about testing, not strictly info for contributors, and too specific for essentials imo. It isn't about Base.Test but i think the docs belong here in a separate section header regardless.